node-fs-readfile/read-any.js

```diff
@@ -0,0 +1,5 @@
+const fs = require('fs');
+fs.readFile('../node-fs-readfile/' + process.argv[2], 'utf8', (err, data) => {
```

Comment:
You don't need to go up a folder then back into it, you can just directly use the filename